packages/connectivity/src/scp-cf/jwt.spec.ts

```diff
@@ -107,13 +105,15 @@ describe('jwt', () => {
     it('fails for no key', async () => {
       nock(jku).get('/').reply(200, { keys: [] });
 
-      await expect(() =>
-        verifyJwt(signedJwtForVerification(jwtPayload, jku))
+      await expect(
+        verifyJwt(signedJwtForVerification(jwtPayload, jku), {
+          cacheVerificationKeys: false
+        })
       ).rejects.toMatchObject({
-        message: 'Failed to verify JWT. Could not retrieve verification key.',
+        message: 'Failed to verify JWT.',
         cause: {
           message:
-            'No verification keys have been returned by the XSUAA service.'
+            'Obtained token keys from UAA, but key with requested keyID "https://my-jku-url.authentication.sap.hana.ondemand.comkey-id-1" still not found in cache.'
```

Comment:
I know this comes from xssec, but if I disable the cacheVerificationKeys, and the error message still says the key is not found in cache, I would be confused.